### ISSUE TITLE
License checking

### ISSUE THREAD

**Issue by peonor** [ericchansen/q2mm#24]
I've had cases now where licenses have been available, but the license checking in the Q2MM code itself fails for various reasons. I'd like a modification so that only proof of licenses unavailable will break here; on checking failure, let the code continue and keep your fingers crossed.

**Comment by ericchansen** [comment]
Do you want something like this? If the regex fails, go for it anyway. This may lead to data misalignment because MacroModel doesn't always return a nonzero exit status when it fails to complete a job, but I suppose it would fail either way.

**Comment by peonor** [comment]
The only thing that would worry me in this case is if there is an old log file from a previous iteration around, that got used instead of the one that should have been created. As long as the data misalignment is guaranteed to cause a crash, not erroneous data, I’d say let it run.

**Comment by ericchansen** [comment]
Oh, good thinking. Maybe the simplest way to deal with this would be to delete *.q2mm.* files before attempting a new MacroModel calculation.

**Comment by peonor** [comment]
Yes. It's not thinking, it's remembering (I've done sooo many mistakes in this field...)
